Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -333,6 +354,7 @@ extension CarPlayManager: CPApplicationDelegate {
         interfaceController.setRootTemplate(mapTemplate, animated: false)
             
         eventsManager.sendCarPlayConnectEvent()
+        os_log("CarInterfaceController did connect to window.", log: logger, type: .info)
```

Comment:
I think that `CPApplicationDelegate` delegate methods are deprecated. We'll need similar logging for `CPTemplateApplicationSceneDelegate` as well. I think that it'd be also beneficial to log information regarding `CPInterfaceController` (e.g. its address, main template etc).